src/db.c

```diff
@@ -57,36 +57,51 @@ void updateLFU(robj *val) {
  * The histogram is a base-2 logarithmic histogram, with 64 bins. The i'th bin 
  * represents the number of keys with a size in the range 2^i and 2^(i+1) 
  * exclusive. oldLen/newLen must be smaller than 2^48, and if their value 
- * equals 0, it means that the key is being created/deleted, respectively. Each 
+ * equals -1, it means that the key is being created/deleted, respectively. Each
```

Comment:
If oldLen or newLen is -1, it seems like this function is NOOP. Am I missing something?

Reply:
okay, I get it now, it makes sense when either one is -1